cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -478,7 +478,7 @@ const liteTestSuites = canonicalTestSuites
     );
 
     // All lite enabled pages should have the LiteSiteCTA component
-    liteSiteTests.push(assertLiteSiteCTAComponentClick);
+    liteSiteTests.push(assertLiteSiteSummaryComponentToMainSiteClick);
```

Comment:
Can we also update this comment now - `    // All lite enabled pages should have the Lite Site Summary component`